src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
-export interface Figure {
+type Colors = 'red' | 'green' | 'blue';
+
+export enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
 
+export interface Figure {
+  shape: Shapes.Triangle | Shapes.Circle | Shapes.Rectangle,
+  color: Colors,
+  getArea(): number,
 }
 
 export class Triangle {
+  shape: Shapes = Shapes.Triangle;
+
+  constructor(
+    public color: Colors,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c].sort((firstE, secondE) => (
+      secondE - firstE
+    ));
+
+    if (sides[0] >= sides[1] + sides[2]) {
+      throw new Error('Error');
+    }
 
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Error');
+    }
+    this.color = color;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+
+    return Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
```

Comment:
In the Triangle class, the getArea method does not round down the area to the nearest hundredths. This operation should be done in the getInfo function to follow the task description.